.circleci/config.yml

```diff
@@ -67,7 +67,19 @@ jobs:
       # copy the circleci config
       - run: sudo cp tests/config-ci.php config.php
       - run: sudo apt-get update
-      - run: sudo DEBIAN_FRONTEND=noninteractive apt-get install -y graphicsmagick libgraphicsmagick1-dev libldap2-dev brotli zopfli
+      - run: sudo DEBIAN_FRONTEND=noninteractive apt-get install -y libldap2-dev brotli zopfli
+      # build imagemagick from source
```

Comment:
is this still needed if the tests are run in docker images?

Reply:
So far I only changed scrutinizer to use docker.
I can try CircleCI next.
Ideally there would be a common docker image for CI. Maybe the next step.
Anyways, why are both tools used, Scrutinizer and CircleCI? Only for historic reasons? To me it seems they do nearly the same thing.

Reply:
Yes, circleci can be done after, with the same image. Both are used because scrutinizer brings bug detection with their software, and circleci brings coverage and more importantly a nice green badge for the README! :p But yes, you are right they do the more or less the same and only scrutinizer could be kept, but I want to keep circleci at least for the time being.

Reply:
BTW I didn't realize this was a circleci.yml file..... -_-'